averbis/core/_rest_client.py

```diff
@@ -1404,6 +1428,76 @@ def upload_resources(
         return self.client._upload_resources(zip_file, project_name=self.name)["files"]
 
 
+class EvaluationConfiguration:
+    def __init__(
+        self,
+        comparison_annotation_type_name: str,
+        features_to_compare: List[str],
+        reference_annotation_type_name: str = None,
+    ):
+        """
+        Configuration for the evaluation of one annotation type
+
+        :param comparison_annotation_type_name:   fully qualified name of the annotation that will be compared;
+            can also be a rule of format fully_qualified_name[feature1=value1&amp;&amp;feature2=value2...]
+            can be extended by another rule of the same type, meaning that an annotation must be contained, e.g:
+            fully_qualified_name[feature1=value1&amp;&amp;feature2=value2...] >
+            fully_qualified_name[feature1=value1&amp;&amp;feature2=value2...]
+        :param reference_annotation_type_name:   fully qualified name of the annotation in the reference text analysis
+            result that other annotations should be compared to; can also be a rule (see annotation_type_name above).
+            If not given, the same annotation as the comparison annotation is used
+        :param features_to_compare:   The list of features that should be used in the comparison, e.g., begin, end,
+            uniqueID.
+        """
+        self.partialMatchCriteria: Union[str, None] = None
+        self.partialMatchArguments: List[str] = []
+        # Features to be excluded from deep feature structure comparisons. These are regular
+        # expressions which match against the fully qualified feature name (type:feature).
+        self.excludeFeaturePatterns: List[str] = []
+        # Regular expression specifying character sequences that should be ignored when
+        # values of string features are compared
+        self.stringFeatureComparisonIgnorePattern = None
+        self.compareAnnotationRule = comparison_annotation_type_name
+        if reference_annotation_type_name:
+            self.goldAnnotationRule = reference_annotation_type_name
+        if reference_annotation_type_name is None:
+            self.goldAnnotationRule = comparison_annotation_type_name
+        self.featuresToBeCompared = features_to_compare
+        self.allowMultipleMatches = False
+        self.stringFeatureComparisonIgnoreCase = False
+        self.forceComparisonWhenGoldstandardMissing = False
+
+    def add_feature(self, feature_name: str) -> None:
+        self.featuresToBeCompared.append(feature_name)
+
+    def set_overlap_partial_match(self) -> None:
```

Comment:
A `set` method should take a value that is the value to be set. In this case, it should be a boolean that enables or disables partial overlap matching. Since this is not a setter, maybe rename to `use_XXX`?